testsuite/MDAnalysisTests/coordinates/test_crd.py

```diff
@@ -66,11 +67,36 @@ def CRD_iter(fn):
         for ref, other in zip(CRD_iter(CRD), CRD_iter(outfile)):
             assert ref == other
 
-    def test_write_EXT(self):
-        # TODO: Write tests that use EXT output format
-        # Must have *lots* of atoms, maybe fake the system
-        # to make tests faster
-        pass
+    def test_write_EXT(self, u, outfile):
+        # Use the `extended` keyword to force the EXT format
+        u.atoms.write(outfile, extended=True)
+
+        with open(outfile, 'r') as inf:
+            format_line = inf.readlines()[2]
+            assert 'EXT' in format_line, "EXT format expected"
```

Comment:
can be outside context manager
```suggestion
        assert 'EXT' in format_line, "EXT format expected"
```